.env

```diff
@@ -0,0 +1,7 @@
+FLASK_ENVIRONMENT_CONFIG=[(dev/prod/test)]
```

Comment:
this file in on `.gitignore` file, therefor it shouldn't be committed 🤔 
Can you please remove this from your PR, because then people won't be mistaken about committing with sensitive data.